test/jdk/java/util/logging/FileHandlerAccessTest.java

```diff
@@ -0,0 +1,113 @@
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+/**
+ * @test
+ * @bug 8252883
+ * @summary tests the handling of AccessDeniedException due to delay in Windows deletion.
+ * @modules java.logging/java.util.logging:open
+ * @run main/othervm FileHandlerAccessTest process 20
+ * @run main/othervm FileHandlerAccessTest thread 20
+ * @author evwhelan
+ */
+
+public class FileHandlerAccessTest {
+    public static void main(String[] args){
+        if (!(args.length == 2 || args.length == 1)) {
+            System.out.println("Usage error: expects java FileHandlerAccessTest [process/thread] <count>");
+            System.exit(1);
+        }
+        else if (args.length == 2) {
+            var type = args[0];
+            var count = Integer.parseInt(args[1]);
+
+            for (var i = 0; i < count; i++) {
+                System.out.println("Testing with arguments: type=" + type + ", count="+count);
+                if (type.equals("process")) {
+                    new Thread(FileHandlerAccessTest::accessProcess).start();
+                }
+                else if (type.equals("thread")) {
+                    new Thread(FileHandlerAccessTest::access).start();
+                }
+            }
+        }
+        else {
+            access();
+        }
+    }
+
+    private static void access(){
+        try {
+            var handler = new FileHandler("sample%g.log", 1048576, 2, true);
+            handler.publish(new LogRecord(Level.SEVERE, "TEST"));
+            handler.close();
+        }
+        catch(Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void accessProcess() {
+        final var javaHome = System.getProperty("java.home");
+        var name = Thread.currentThread().getName();
+        BufferedReader bufferedReader = null;
+        Process childProcess = null;
+        final String className = new Object(){}.getClass().getEnclosingClass().getName();
+
+        try {
+            ProcessBuilder processBuilder = new ProcessBuilder(javaHome + File.separator + "bin" + File.separator + "java", "-cp", ".", className, "doProcess");
+            processBuilder.redirectErrorStream(true);
+            childProcess = processBuilder.start();
+
+            bufferedReader = new BufferedReader(new InputStreamReader(childProcess.getInputStream()));
+            String line;
+            while((line = bufferedReader.readLine()) != null) {
+                System.out.println(name + "\t|" + line);
+            }
+            childProcess.waitFor();
+        }
+        catch(Exception e) {
+            e.printStackTrace();
```

Comment:
Same remark here: should this make the test fail?

Reply:
Thanks Daniel, please see my above reply